src/components/KAlert/KAlert.vue

```diff
@@ -130,7 +130,7 @@ import type { PropType } from 'vue'
 import { computed, useSlots } from 'vue'
 import KButton from '@/components/KButton/KButton.vue'
 import KIcon from '@/components/KIcon/KIcon.vue'
-import type { AlertAppearance, AlertDismissType, AlertAppearanceRecord, AlertSize, AlertType } from '@/types'
+import { type AlertAppearance, type AlertDismissType, type AlertSize, type AlertType, AlertAppearances } from '@/types'
```

Comment:
This type import format is incorrect - why is the eslint rule not catching this?

Reply:
```suggestion
import type { AlertAppearance, AlertDismissType, AlertSize, AlertType } from '@/types'
import { AlertAppearances } from '@/types'
```